### PR TITLE
1989 emite resultado proc seletivo

### PR DESCRIPTION
**Pull request by robertojnior**

### CODE REVIEW

File: aplicacoes/gpa-mae/src/main/webapp/WEB-INF/spring-security.xml [hunk 1]
```diff
@@ -16,6 +16,9 @@
  		<intercept-url pattern="/selecao/listar" access="permitAll"/>
  		<intercept-url pattern="/selecao/downloadDocumento/*" access="permitAll"/>
  		
+<<<<<<< HEAD
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/WEB-INF/spring-security.xml]: resolver conflito

File: aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp [hunk 2]
```diff
@@ -204,6 +204,15 @@
 													Avaliar Inscritos <span class="glyphicon glyphicon-user"></span>
 												</button>
 											</a>
+<<<<<<< HEAD
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp]: resolver conflito

File: aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp [hunk 3]
```diff
@@ -211,6 +220,7 @@
 											href="<c:url value="/selecao/informacoes/${selecao.id}"></c:url>">
 											<button class=" btn btn-success">
 												+ Informações <span class="glyphicon glyphicon-zoom-in"></span>
+>>>>>>> d5a3b5d8926b79a82ba9cbc147ac76698fd3f755
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/WEB-INF/views/selecao/listar.jsp]: resolver conflito

File: aplicacoes/gpa-mae/src/main/webapp/resources/css/estilo.css [hunk 4]
```diff
@@ -108,7 +108,15 @@ input[type=checkbox] {
 	padding: 0%;
 }
 
+<<<<<<< HEAD
```
Comment (regispires) [inline aplicacoes/gpa-mae/src/main/webapp/resources/css/estilo.css]: resolver conflito

**Comment by robertojnior** [comment]
Conflitos resolvidos

**Pull request merged**